app/views/layouts/admin.haml

```diff
@@ -0,0 +1,29 @@
+!!!
+%html{lang: I18n.locale}
+  %head
+    %meta{content: 'text/html; charset=UTF-8', 'http-equiv' => 'Content-Type'}
+    %title= admin_title.html_safe
+    %meta{charset: 'utf-8'}
+    %meta{content: 'width=device-width, initial-scale=1 shrink-to-fit=no', name: 'viewport'}
+    %meta{content: admin_description.html_safe, name: 'description'}
+    %link{rel: 'shortcut icon', href: @current_community.favicon}
+
+  - path_font = APP_CONFIG[:font_circularstd_url]
+  - if path_font.present?
+    = render 'layouts/fonts_admin_v2', path_font: path_font
+
+  = stylesheet_link_tag 'admin2/admin'
+  = javascript_include_tag 'admin2/admin'
+  %body
+    = render partial: 'admin2/mobilemenu', locals: header_props
+    #root
+      = render partial: 'admin2/topnav', locals: header_props
+      = flash_messages
+      .layout-container
+        = render 'admin2/sidebar'
+        .content-card
+          = yield
+    .mobilemenu-overlay
+    %script{src: 'https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.3/umd/popper.min.js'}
+    %script{src: 'https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/js/bootstrap.min.js'}
```

Comment:
It would be better if these are not loaded from external URLs, but added to the assets in the source in `vendor/assests/javascripts`. It's safer that way, as we wouldn't be loading scripts from (untrusted) external URLs by default, as well as we wouldn't depend on the external service to be accessible, as we'd serve the scripts from our own infra.

Reply:
done!